src/Maker/MakeResetPassword.php

```diff
@@ -371,8 +372,14 @@ private function generateRequestEntity(Generator $generator, ClassNameDetails $r
 
         $generator->writeChanges();
 
+        $useAttributesForDoctrineMapping = $this->doctrineHelper->isDoctrineSupportingAttributes() && $this->doctrineHelper->doesClassUsesAttributes($requestClassNameDetails->getFullName()) && Kernel::VERSION_ID >= 50200;
```

Comment:
Same comment. You should not rely on version of 1 component.
Instead you should check that the classes you need are present.